client/config/config.go

```diff
@@ -124,7 +124,7 @@ func (c *Config) GrpcClientConfig() (grpc.ClientConfig, error) {
 	return grpc.ClientConfig{
 		ServerAddress: srvAddr,
 		Token:         c.Token,
-		Insecure:      hasInsecureScheme(c.GrpcURL),
+		Insecure:      c.InsecureGRPC,
```

Comment:
It's a missing change from last improvements on docker-compose to allow client to connect insecurely.